src/components/HomePage/SearchResultsArea/Offer/Offer.js

```diff
@@ -1,10 +1,11 @@
 export default class Offer {
-    constructor({ id, position, company, location, date, description }) {
+    constructor({ id, position, company, location, date, description, fields }) {
         this.id = id;
         this.position = position;
         this.company = company;
         this.location = location;
         this.date = date;
         this.description = description;
+        this.fields = fields;
     }
 }
```

Comment:
Don't edit this file in this PR, it's already done in #78, and it is more complete than this